.github/workflows/doc-index-updater-branch.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - .github/workflows/doc-index-updater-branch.yaml
 
 env:
-  IMAGE: mhraproductsnonprodregistry.azurecr.io/products/doc-index-updater
+  IMAGE: mhraproductsdevelopmentregistry.azurecr.io/products/doc-index-updater
```

Comment:
did you intent to use the dev registry? I thought we were only going to use the non-prod one ?

Reply:
I did intend this. I don't remember that, although it sounds reasonable as the tags shouldn't clash. If we're intending to only use non-prod one, should we remove the existing development one?